utils/json/int64.go

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package json
+
+import "strconv"
+
+type Int64 int64
+
+func (i Int64) MarshalJSON() ([]byte, error) {
+	return []byte(`"` + strconv.FormatInt(int64(i), 10) + `"`), nil
+}
```

Comment:
q: should we guarantee that for all int64 values, `json.Uint64` and `json.Int64` are interchangeable?
Otherwise we are changing a bunch of APIs right?

Reply:
I'm not sure what you mean by interchangeable -- there are some values that can be expressed in one but not the other. Do you mean testing that specifically, for all `uint64` that are `< math.MaxInt64`, that the marshalled `int64` and `uint64` representation are the same?